### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by Blessman-Newton**
Add Navbar containing font icons
Add a Hero section including hero text, font icons, and background image
Add work section 
Add frameworks, skills, and language 
Add contact form with validation prompt
Add a footer with font icons
Style.css file modify
index.html file modify

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,21 +19,31 @@
...
+                    <h1>Hey There. <br> I'm Blessman.</h1>
+                    <h2>I'm a Software Developer</h2>
+                    <p class="text">
+                        I can help you build a product , feature
+                        or website Look through some of my 
+                        work and experience! If you like what you 
+                        see and have a project you need coded, 
+                        don’t hesitate to contact me.
+                    </p>
+                </div>
```
Comment (Qoosim) [inline index.html:46]: - [ ] It is required to stick to the figma template as much as possible. There is a missing background image on your design for the headline section. Kindly add the proper background-image to this section.

### Your Design
your-headline-desktop

### Figma Design
temp-headline-desktop

File: index.html [hunk 2]
```diff
@@ -50,11 +60,15 @@ <h2>I'm a Software Developer</h2>
...
-                <hr>
+                <h3>My Recent Works</h3>
+                <hr class="recent-hr">
             </div>
             <div class="project-box">
-                <img src="images/Img Placeholder.png" alt="">
+                <div class="img-holder">
+                    <img id="img1" src="images/Img Placeholder.png" alt="">
+                    <img id="img2" src="images/Img Placeholder (3).png" alt="">
+                </div>
```
Comment (Qoosim) [inline index.html:70]: - [ ] It is required to stick to the figma template as much as possible. Your design for your first project and the text my recent works with the horizontal line does not look like how it is on the figma design. The image on the right on your project card does not show. Kindly walk your design alongside the figma design. 

### Your Design
your-recent-work

### Figma Design
temp-recent-work

File: index.html [hunk 3]
```diff
@@ -50,11 +60,15 @@ <h2>I'm a Software Developer</h2>
...
             <div class="project-box">
-                <img src="images/Img Placeholder.png" alt="">
+                <div class="img-holder">
+                    <img id="img1" src="images/Img Placeholder.png" alt="">
+                    <img id="img2" src="images/Img Placeholder (3).png" alt="">
+                </div>        
+                <div class="work-content">
                     <h2>Multi-Post-Stories</h2>
                     <p>
                         A daily selection of privately personalized
```
Comment (Qoosim) [inline index.html:74]: - [ ] It is required to stick to the figma template as much as possible. Your projects list under this section does not show the complete projects list. Kindly look into it and fix.

### Your Design Desktop
your-project-lists

### Figma Design Desktop
temp-project-lists


### Your Design Mobile
your-project-card

### Figma Design Mobile
temp-project-card-mobile

File: index.html [hunk 4]
```diff
@@ -75,112 +89,167 @@ <h2>Multi-Post-Stories</h2>
...
+                    Hello I’m a software developer! I can help <br>
+                    you build a product , feature or website <br>
+                    Look through some of my work and <br> 
+                    experience! If you like what you see and 
+                    <br> have a project you need coded, don’t 
+                    <br> hesitate to contact me.
+                </p>
+                <div class="resume-btn">
+                    <button>Get my resume</button>
+                </div>
```
Comment (Qoosim) [inline index.html:352]: - [ ] The backgroud image on the figma is missing on your design. Kindly look into it and fix.

### Your Design
your-about-me

### Figma Design
temp-about-me

File: index.html [hunk 5]
```diff
@@ -75,112 +89,167 @@ <h2>Multi-Post-Stories</h2>
...
-        <hr>
-        <div class="about_footer">
+           <hr>
+         <div class="about_footer">
             <div class="profile">
-               <ul>
+               <ul class="ul">
                    <li class="list-style Language">Language
                        <ul>
                             <li>JavaScript</li>
```
Comment (Qoosim) [inline index.html:360]: - [ ] Kindly look into your style for all the symbols and how they're styled on the figma design and ensure you produce a close replica of each of them on your design.

### Your Design
your-lang

### Figma Design
temp-lang

File: index.html [hunk 6]
```diff
@@ -224,12 +293,12 @@ <h1>About me</h3>
...
                 <input class="input" type="email" name="email" placeholder="Email address" required>
-                <textarea name="send" maxlength="500" required placeholder="Write me something"></textarea>
+                <textarea name="send" maxlength="399" required>Write me something</textarea>
                 <input class="submit" type="submit" value="Get in touch">
             </div>
         </form>
     </section>
-    <hr>
+    <hr class="hr">
     <footer class="footer">
```
Comment (Qoosim) [inline index.html:408]: - [ ] Kindly adjust the text and the form alignment on your design to match the figma design.
- [ ] You also need to add the social media lists that are missing at the bottom of your design.

### Your Design Desktop
your-form-desktop

### Figma Design Desktop
temp-form-desktop

- [ ] You also need to work on the style for the social media on the mobile.

### Your Design
[app-behave-form.webm](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/6eb3967a-06e2-429b-9702-f11b558325eb)


### Figma Design
temp-form

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Blessman-Newton, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @Blessman-Newton , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Nice Design ✔️ 
- All requested changes implemented ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**